src/edtr-io/plugins/paste-hack/editor.tsx

```diff
@@ -0,0 +1,61 @@
+
+export const PasteHackEditor: React.FunctionComponent<PasteHackPluginProps> = (
+  props
+) => {
+  const store = useScopedStore()
+
+  function replaceWithStateString() {
+    console.log('starting replace')
+    const newDocuments: DocumentState[] = []
+
+    const content = JSON.parse(dummyContent) as {
+      plugin: 'rows'
+      state: DocumentState[]
+    }
+    if (content.plugin === 'rows') {
+      content.state.forEach((plugin) => {
+        newDocuments.push(plugin)
+      })
+    } else {
+      newDocuments.push(content)
+    }
+
+    // TODO: check if we could replace this plugin with multiple plugins instead of nesting rows plugins
+    // like the layout plugin does
+    // store.getState()
+    // get parent rows plugin:
+    //const parent = getParent(props.id)(store.getState())
+
+    store.dispatch(
+      replace({
+        id: props.id,
+        plugin: 'rows',
+        state: newDocuments,
+      })
+    )
```

Comment:
Here you create a nested row -> we need an implementation where we replace this plugin with multiple other plugins...

Reply:
Yeah, that's one of the questions I asked jonas and then you. Any ideas?

Reply:
https://serlo.slack.com/archives/D0H3UB1E2/p1666600535956429

Reply:
Here is my idea, which now seems to work: https://github.com/serlo/frontend/pull/1850/commits/a6543689ea29c800008658e27daa8a943a075478